src/cmd/services/m3coordinator/downsample/downsampler_test.go

```diff
@@ -1008,6 +1088,21 @@ func TestDownsamplerAggregationWithRulesConfigRollupRulesPerSecondSum(t *testing
 		},
 	})
 
+	// Setup auto-mapping rules.
+	require.False(t, testDownsampler.downsampler.Enabled())
```

Comment:
In addition to the explicit test case, I wanted to ensure that another rollup rule test case actually had auto mapping rules setup.